File: sct_source/SourceCode/API_SoCongThuong/Models/CateInvestmentProjectModel.cs
using System;
using System.Collections.Generic;

namespace API_SoCongThuong.Models
{
    public partial class CateInvestmentProjectModel
    {
        public Guid CateInvestmentProjectId { get; set; }
        /// <summary>
        /// 1: trong nước, 2 ngoài nước
        /// </summary>
        public int InvestmentType { get; set; }
        public string BusinessName { get; set; } = null!;
        public string? Owner { get; set; } = "";
        public string? PhoneNumber { get; set; }
        public string? Career { get; set; }
        public long Investment { get; set; }
        public int NumberOfWorker { get; set; }
        /// <summary>
        /// Diện tích dự án
        /// </summary>
        public int ProjectArea { get; set; }
        /// <summary>
        /// Sản lượng 1 ngày
        /// </summary>
        public int Quantity { get; set; }
        /// <summary>
        /// năng suất sản xuất
        /// </summary>
        public int Produce { get; set; }
        public long ProductValue { get; set; }
        public string? Reality { get; set; }
        /// <summary>
        /// 1: Hoạt động 0: Không hoạt động
        /// </summary>
        public bool? IsAction { get; set; }
        /// <summary>
        /// 1: Đã xóa; 0: Chưa xóa
        /// </summary>
        public bool IsDel { get; set; }
        /// <summary>
        /// Thời gian tạo
        /// </summary>
        public DateTime CreateTime { get; set; }
        public string CreateTimeDisplay { get; set; } = "";
        /// <summary>
        /// Người tạo
        /// </summary>
        public Guid CreateUserId { get; set; }
        public string CreateName { get; set; } = "";
        public DateTime? UpdateTime { get; set; }
        public Guid? UpdateUserId { get; set; }
        public Guid? District { get; set; } = Guid.Empty;
    }
}

File: sct_source/SourceCode/FE_SoCongThuong/Angular/src/app/modules/app-pages/industrial-cluster-info-management-page/_models/cate-investment-project.model.ts
import { BaseModel } from '../../../../_metronic/shared/crud-table/models/base.model';

export interface CateInvestmentProjectModel extends BaseModel {
  cateInvestmentProjectId : string;
  investmentType: number | null; //Loại đầu tư 1. Trong nước / 2. Ngoài nước
  businessName : string; //Tên doanh nghiệp
  investment : number | null; //Vốn đăng ký
  numberOfWorker: number | null; //Số lượng nhân viên
  projectArea: number | null; //Diện tích dự án
  quantity: number | null; //Sản lượng (SP / ngày)
  produce: number | null; //Công suất
  productValue: number | null;//Giá trị sản phẩm
  reality: string; //Thực trạng
  owner: string; // chủ doanh nghiệp
  phoneNumber: string;
  career: string; //ngành nghề
  district: string;
}
